packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/views/diagramfilter/services/api/IDiagramFilterSplitButtonContributionProvider.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.application.views.diagramfilter.services.api;
+
+import org.eclipse.sirius.components.forms.description.ButtonDescription;
+
+/**
+ * Provides the description of a button for the diagram filter's split button.
+ *
+ * @author gdaniel
+ */
+public interface IDiagramFilterSplitButtonContributionProvider {
```

Comment:
May be this could be renamed to something that is not related to the SplitButton, like `IDiagramFilterActionContributionProvider`? What do you think about?

Reply:
done